transitions/core.py

```diff
@@ -334,6 +337,7 @@ def __init__(self, model=None, states=None, initial=None, transitions=None,
                 executed in a state callback function will be queued and executed later.
                 Due to the nature of the queued processing, all transitions will
                 _always_ return True since conditional checks cannot be conducted at queueing time.
+            add_self (boolean): If no model(s) provided, intialize state machine against self.
```

Comment:
Since a `model=None` currently adds `self` to the list I don't understand why you wouldn't also just want that functionality here.